lms/static/scripts/frontend_apps/utils/hooks.ts

```diff
@@ -44,8 +44,13 @@ export function usePlaceholderDocumentTitleInDev() {
       return () => {};
     }
 
-    const listener = () => {
-      document.title = 'PLACEHOLDER DOCUMENT TITLE';
+    const listener = (e: NavigateEvent) => {
+      const { pathname: destination } = new URL(e.destination.url);
+      const currentUrl = location.pathname;
```

Comment:
```suggestion
      const currentPath = location.pathname;
```

This would be `currentURL` using our naming convention, but here the value is a path rather than a URL.